roles/agnosticv/templates/template.yml.j2

```diff
@@ -49,5 +49,24 @@ objects:
           vars: {}
 {% endif %}
 
-parameters: {{ merged_vars.__meta__.catalog.parameters | d([]) | to_json }}
+{% if merged_vars.__meta__.catalog.parameters is defined %}
+parameters:
+{%   for parameter in merged_vars.__meta__.catalog.parameters %}
+- name: {{ parameter.name | to_json }}
+{%     if parameter.description is defined %}
+  description: {{ parameter.description | to_json }}
+{%     endif %}
+{%     if parameter.formLabel is defined %}
+  displayName: {{ parameter.formLabel | to_json }}
```

Comment:
any reason why not use `formLabel` or `displayName` in both places, in catalog_item and in agnosticv definition ?

Reply:
`displayName` is what the OpenShift template requires, but `formLabel` is more accurate, particularly when it comes to checkbox groups where we will want both a `formGroup` and a `formLabel`... hrm. The `formGroup` should be included in the `displayName` when it is set...